Update Blazor debug for hosted config and Edge (#19213)
dotnet/AspNetCore.Docs: aspnetcore/blazor/debug.md

}
```

#### Launch and debug a hosted Blazor WebAssembly app with Microsoft Edge

> Browser configuration defaults to Google Chrome. When using Microsoft Edge for debugging, set `browser` to `edge`. To use Google Chrome, don't set `browser` or set it to `chrome`.

Reviewer: ```suggestion
Browser configuration defaults to Google Chrome. When using Microsoft Edge for debugging, set `browser` to `edge`. To use Google Chrome, unset the `browser` property in the configuration.
```

Author: It's not config that the dev would unset if it weren't there in the first place. Also, "unset" (or even "don't set" in the original come to think of it), makes it sound like an empty string is a valid value, which it isn't.

```suggestion
Browser configuration defaults to Google Chrome. When using Microsoft Edge for debugging, set `browser` to `edge`. To use Google Chrome, either don't set the `browser` option or set the option's value to `chrome`.
```
